roles/multibench_run/tasks/main.yml

```diff
@@ -0,0 +1,55 @@
+---
+# If this shell cmd ahs a correct rc, it means the multibench_host is ready
+- name: "Check the multi-bench host has the crucible binaries"
+  ansible.builtin.shell: >
+    crucible help
+  delegate_to: multibench
+  become: true
+
+# This file contains some information about the execution like the number of iterations, the tags...
+- name: "Generate config file for multibench"
+  ansible.builtin.template:
+    src: templates/config.ini.j2
+    dest: "{{ multibench_script_dir }}/config.ini"
+    mode: '0644'
+  delegate_to: multibench
+  become: true
+
+- name: "Creating an output directory if needed"
+  when:
+    - multibench_run_output_dir is undefined
```

Comment:
should `multibench_run_output_dir.path` be checked too? since it's the variable that is eventually checked?